capstone-app/src/app/modules/photos/components/image-container/image-container.component.ts

```diff
@@ -1,39 +1,66 @@
 import { Photo } from 'src/app/modules/photos/photo-template';
 import { PhotoFetcher } from 'src/app/modules/photos/services/photo-fetcher.service';
-import { Component, OnInit, Input, OnChanges, Output, EventEmitter } from '@angular/core';
+import {
+  Component,
+  Input,
+  OnChanges,
+  Output,
+  EventEmitter,
+  OnInit,
+} from '@angular/core';
 
 @Component({
   selector: 'app-image-container',
   templateUrl: './image-container.component.html',
   styleUrls: ['./image-container.component.scss'],
 })
-export class ImageContainerComponent implements OnInit, OnChanges  {
+export class ImageContainerComponent implements OnChanges, OnInit {
   displayPhotos: Photo[] = [];
   originalPhotos: Photo[] = [];
-  query: string;
   errorMessage: string;
+  query = '';
+  previousCity = '';
+  previousFilter = '';
 
   /**
-   * @Input limit - Receives count filter from parent on how many
-   * photos to display
-   * @Output limitChange - When an image cannot be displayed,
-   * tell the parent to update image count
+   * @Input city: the city user searched for.
```

Comment:
jsdoc is typically used once for each property. So for example, instead of writing it like this, it should be

```
/** Comment here */
@Input city: string;

/** Another comment here */
@Input filter: string;
```

Also, it is only necessary when the comment adds some additional information that the variable name itself does not provide. I'd argue that "city" and "filter" don't really need a jsdoc. Limit might need one, but this could easily be fixed by just calling it "photoLimit" instead.